settings/DevHome.Settings/Views/AboutPage.xaml

```diff
@@ -2,29 +2,35 @@
     x:Class="DevHome.Settings.Views.AboutPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:behaviors="using:DevHome.Common.Behaviors"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:ui="using:CommunityToolkit.WinUI"
-    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
-    mc:Ignorable="d">
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:viewmodels="using:DevHome.Settings.ViewModels"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    behaviors:NavigationViewHeaderBehavior.HeaderContext="{x:Bind ViewModel}">
+
+    <behaviors:NavigationViewHeaderBehavior.HeaderTemplate>
+        <DataTemplate x:DataType="viewmodels:AboutViewModel">
+            <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
+                <BreadcrumbBar
+                    x:Name="BreadcrumbBar"
+                    ItemsSource="{x:Bind Breadcrumbs}">
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="ItemClicked">
+                            <ic:InvokeCommandAction Command="{x:Bind BreadcrumbBarItemClickedCommand}" />

```

Comment:
Instead of using the `EventTriggerBehavior`, we can implement our own custom `Behavior` which would host the logic of `ItemClicked` inherited by the derived classes